flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/fetcher/SplitFetcherManager.java

```diff
@@ -238,6 +274,15 @@ public boolean maybeShutdownFinishedFetchers() {
         return fetchers.isEmpty();
     }
 
+    /**
+     * Return the queue contains data produced by split fetchers.This method is Internal and only
+     * used in {@link SourceReaderBase}.
+     */
+    @Internal
+    public FutureCompletingBlockingQueue getQueue() {
```

Comment:
It's better to specify the type parameter of `FutureCompletingBlockingQueue`, which should be `RecordsWithSplitIds<E>`